.agent/workflows/develop.md

```
@@ -0,0 +1,212 @@
+Follow the Development Loop from `basic-agentic-development`:
+
+### For each PBI (in planned order):
+
+1. **Discovery** — Read existing code in affected areas (`view_file_outline` first, then targeted reads).
+2. **Implement** — Make changes incrementally, smallest testable unit first.
+3. **Follow conventions** — Use existing patterns from the codebase:
+   - Attribute-based DI (`[Scoped]`, `[Singleton]`, etc.)
+   - Source-Known ID pattern for entities
+   - Existing namespace conventions (verify with `grep_search`, never assume)
+4. **Validate after each change**:
+   ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)